app/src/main/java/com/celzero/bravedns/ui/AppConnectionBottomSheet.kt

```diff
@@ -78,11 +92,20 @@ class AppConnectionBottomSheet(private val adapter: AppConnectionAdapter?, val u
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        uid = arguments?.getInt(UID) ?: -1
```

Comment:
don't use magic ints (`-1` here). I believe there's already a `MISSING_UID` / `UNKNOWN_UID` or some such declared?

Reply:
Modified to use proper variable name.